### DISCUSSION TITLE
Phi axis direction change for ODF figures

### DISCUSSION THREAD

**Discussion by YAP787** [mtex-toolbox/mtex#2252] [Ask Anything]
Hi,

In the ODF figures I want to keep the Phi [0 90] pointing down. But no matter I changed the settings in PhiSections.m or set in gca, both didn't work. Can anyone suggest why is that? Thanks.
![image](https://github.com/user-attachments/assets/f54fcfaf-dda0-4477-9fcf-f344ecabfa66)
![image](https://github.com/user-attachments/assets/8d40980b-6280-4bb9-b63a-8eaf624db5db)
![image](https://github.com/user-attachments/assets/04cad890-9dc2-4b83-8b78-d12523987988)

**Comment by ralfHielscher** [comment]
What version of MTEX are you using?

Ralf.

**Reply by YAP787** [reply]
I am using mtex-6.0.beta2.
